erizo/src/erizo/thread/ThreadPool.h

```diff
@@ -0,0 +1,50 @@
+#ifndef ERIZO_SRC_ERIZO_THREAD_THREADPOOL_H_
+#define ERIZO_SRC_ERIZO_THREAD_THREADPOOL_H_
+
+#include <memory>
+#include <vector>
+
+#include "thread/Worker.h"
+
+namespace erizo {
+
+class ThreadPool {
```

Comment:
I generally don't like having the full implementation in the header file. I know workers are already that way...
In this case I guess ThreadPool is simple enough that we don't need a cc file but I wanted to make sure this decision was taken consciously. Maybe we should discuss where we want to draw the line because I would prefer this not becoming a trend.